src/v/config/configuration.cc

```diff
@@ -753,6 +753,13 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       1,
       {.min = 1, .oddeven = odd_even_constraint::odd})
+  , minimum_topic_replication(
+      *this,
+      "minimum_topic_replications",
+      "Minimum permitted value of replication factor for new topics",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      std::nullopt,
```

Comment:
would it make more sense to give it a default value of 1? is it non-intuitive that the default nullopt would allow 1 (i'm guessing) but you couldn't explicitly ask for 1? seems like cloud could just set it to three in their configs?

Reply:
My feeling is that it's kind of redundant for it to be a min of 1.  replication factor must be a positive, odd number, which implies a minimum value of 1 at all times.

Reply:
Haha I ran into this too! Not the only one :)

Reply:
Had a very good conversation with Noah about this and he raised an interesting point:

Are we implying that there is no minimum value for RF when this is disabled?  Obviously the minimum for RF is 1, we have a bunch of checks in our code to ensure that, but to the customer reading the documentation does it imply that there is no minimum?  Can this cause any confusion?

Trying to ensure a consistent customer experience and to not create any confusion for them, I think I'm leaning in the direction of this _not_ being an optional and having the min be 1.

@rockwotj as you have made some comments irt to this, what are your thoughts?

Reply:
+100 to that